swagger/v1/api_spec.json

```diff
@@ -1881,285 +1894,8 @@
           }
         }
       },
-      "ECFParticipantDeclarationAttributes": {
```

Comment:
The declaration API spec is being removed as we combine those in our API endpoint, however in the documentation we have separated those to indicate what is required in ECF vs NPQ, do you mind adding those back please in v1/v2?

Reply:
@cwrw sure, I will put them back for now, but these were added manually directly in the `api_spec.json` files and they will be removed again the next time we run the `swaggerize` rake command.

If they need to stick around, then will need [spec/docs](https://github.com/DFE-Digital/early-careers-framework/tree/develop/spec/docs).